esp-wifi/src/esp_now/mod.rs

```diff
@@ -537,12 +540,12 @@ pub struct EspNowReceiver<'d> {
     _rc: EspNowRc<'d>,
 }
 
-impl<'d> EspNowReceiver<'d> {
+impl EspNowReceiver<'_> {
     /// Receives data from the ESP-NOW queue.
     pub fn receive(&self) -> Option<ReceivedData> {
         critical_section::with(|cs| {
             let mut queue = RECEIVE_QUEUE.borrow_ref_mut(cs);
-            queue.dequeue()
```

Comment:
RECEIVE_QUEUE now becomes LIFO?

Reply:
I don't think there is a material change here, pushing to the back and popping the front is a FIFO?